lib/spaceship/tunes/app_analytics.rb

```diff
@@ -0,0 +1,145 @@
+module Spaceship
+  module Tunes
+    class AppAnalytics < TunesBase
+      # @return (Integer) The app ID to grab analytics for
+      attr_accessor :apple_id
+
+      attr_mapping({
+        'adamId' => :apple_id
+      })
+
+      class << self
+        def factory(attrs)
+          return self.new(attrs)
+        end
+      end
+
+      def app_units
```

Comment:
I wonder if all these methods wouldn't be best served with a smart method_missing implementation.

Pseudo code:
```

def app_#{attr} 
=> app_#{attr}_last_7_days

def app_#{attr}_last_#{x}_#{unit}s
=>
  start_t, end_t = send("time_last_#{x}_#{unit}s")
  client.time_series_analytics([apple_id], [to_measure(attr)], start_t, end_t, to_unit(unit))
```